profitbricks/src/main/java/org/jclouds/profitbricks/compute/function/ImageToImage.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.profitbricks.compute.function;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.jclouds.profitbricks.compute.function.DataCenterToLocation.mapLocation;
+
+import java.util.regex.Pattern;
+
+import org.jclouds.compute.domain.Image;
+import org.jclouds.compute.domain.ImageBuilder;
+import org.jclouds.compute.domain.OperatingSystem;
+import org.jclouds.compute.domain.OsFamily;
+
+import com.google.common.base.Function;
+
+public class ImageToImage implements Function< org.jclouds.profitbricks.domain.Image, Image> {
+
+   private static final Pattern HAS_NUMBERS = Pattern.compile(".*\\d+.*");
+
+   @Override
+   public Image apply(org.jclouds.profitbricks.domain.Image from) {
+      checkNotNull(from, "Null image");
+
+      String desc = from.name();
+      OsFamily osFamily = parseOsFamily(desc);
+
+      OperatingSystem os = OperatingSystem.builder()
+	      .description(osFamily.value())
+	      .family(osFamily)
+	      .version(parseVersion(desc))
+	      .is64Bit(is64Bit(desc))
+	      .build();
+
+      return new ImageBuilder()
+	      .ids(from.id())
+	      .name(desc)
+	      .location(mapLocation(from.location()))
```

Comment:
Images are available in all regions by default? (Just asking as the mapLocation function always sets the PROVIDER scope.